lib/batch_loader/graphql.rb

```diff
@@ -45,7 +45,6 @@ def self.wrap_with_warning(batch_loader)
       warn "DEPRECATION WARNING: using BatchLoader.for in GraphQL is deprecated. Use BatchLoader::GraphQL.for instead or return BatchLoader::GraphQL.wrap from your resolver."
       wrap(batch_loader)
     end
-    private_class_method :wrap_with_warning
```

Comment:
On line 8 of this file, we call `BatchLoader::GraphQL.wrap_with_warning`. So we cannot make this a private method.